terraform_exec.sh

```diff
@@ -9,9 +9,9 @@ APPLY=$1
 echo "${ENV}"
 echo "${AWS_REGION}"
 terraform init \
--backend-config="bucket=terraform-state-${ENV}" \
+-backend-config="bucket=project-terraform-state-${ENV}" \
 -backend-config="key=${ENV}/platform-infra.tfstate" \
--backend-config="dynamodb_table=${ENV}-terraform-state-lock-dynamo" \
+-backend-config="dynamodb_table=${ENV}-dev-project-terraform-state-lock-dynamo" \
```

Comment:
you don't have to force the env, it have to be taken reading ${ENV} var

Reply:
reverted back to the previous one.